.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: gradle
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Would it be better to do it once a week? Daily sounds too bold.  I have real trust issues with that.

Reply:
What kind of issues did you encounter?

Reply:
We will discuss it during the guild

Reply:
Our agreement was to keep everything close to code/PR. Any information about encountered issues is welcome, so don't hesitate to post it here, even if you're not sure if it's valid. That's how we learn from each other 🍻